osu.Game/Skinning/SkinImporter.cs

```diff
@@ -132,7 +132,7 @@ private void updateSkinIniMetadata(SkinInfo item, Realm realm)
             {
                 // skins without a skin.ini are supposed to import using the "latest version" spec.
                 // see https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/Graphics/Skinning/SkinManager.cs#L297-L298
-                newLines.Add($"Version: {SkinConfiguration.LATEST_VERSION}");
+                newLines.Add(FormattableString.Invariant($"Version: {SkinConfiguration.LATEST_VERSION}"));
```

Comment:
I have to say that this kind of method call really hurts my mind to think about (ie. how does an outer method affect a string interpolation contained within it).

And yes I get how it works, just saying 🙃